ludwig/explain/captum.py

```diff
@@ -221,4 +320,46 @@ def get_total_attribution(
     if use_global:
         total_attribution /= nsamples
 
-    return total_attribution
+    feat_to_token_attributions = {k: [e for lst in v for e in lst] for k, v in feat_to_token_attributions.items()}
+
+    return total_attribution, feat_to_token_attributions
+
+
+def get_token_attributions(
+    model: LudwigModel,
+    feature_name: str,
+    input_ids: torch.Tensor,
+    token_attributions: torch.Tensor,
+) -> List[List[Tuple[str, float]]]:
+    """Convert token-level attributions to an array of token-attribution pairs of shape [batch_size,
+    sequence_length, 2].
+
+    # Inputs
```

Comment:
nit: Use https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
Made the change!